test/local_integration_test.py

```diff
@@ -33,6 +33,27 @@ def setUpModule():
 
 
 class IntegrationTest(unittest.TestCase):
+    """
+    The integration tests work by first setting the lambdas in test mode (this happens in setUp). This
+    Sets some environment variable in the indexer lambda that change its behavior slightly, allowing
+    it to process modified test notifications differently.
+
+    Next, the test queries Azul for all of the bundles that match a certain prefix. In
+    AzulClient.reindex() these bundle are used to construct fake notifications. These fake notifications
+    Have an extra information embedded. They include:
+     - the name of the integration test run,
+     - a new uuid for the project, and
+     - a new unique bundle UUID.
+    When the bundle is processed by the indexer lambda the original bundle uuid is used to fetch the
+    metadata, etc., but these new fields are what's actually stored in Azul. This effectively creates a
+    copy of the bundles, but with a new uuid and shared project.
```

Comment:
```suggestion
    copy of the bundles, but each copy with a new and different bundle UUID and all copies sharing a synthesized project with a different project UUID. Note that unlike real projects, the shared project will contain a mix of metadata graph shapes since the bundles that contribute to it are selected randomly.
```